src/bin/cargo/commands/tree.rs

```diff
@@ -42,7 +42,14 @@ pub fn cli() -> Command {
             "SPEC",
             "Prune the given package from the display of the dependency tree",
         ))
-        .arg(opt("depth", "Maximum display depth of the dependency tree").value_name("DEPTH"))
+        .arg(
+            opt(
+                "depth",
+                "Maximum display depth of the dependency tree, \
+                or --depth=worktree to display dependencies from workspace members only",
```

Comment:
Like with `--jobs default`, I wonder if we should leave this for `cargo help tree`

`--jobs default` is less common but not thrilled with how long this has made the help.  There are things we could do to trim it some.